docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -165,7 +156,7 @@ export const pageQuery = graphql`
 
 Now we've got a template. Great! I'll assume you followed the tutorial for [Adding Markdown Pages](/docs/adding-markdown-pages/) and provide a sample `createPages` that generates post pages as well as tag pages. In the site's `gatsby-node.js` file, include `lodash` (`const _ = require('lodash')`) and then make sure your [`createPages`](/docs/node-apis/#createPages) looks something like this:
 
-```js:title=gatsby-node.js
+```js
```

Comment:
Any reason to delete the title of the file? People benefit from this small metadata as it helps them navigate through the source and making sure they are in the right file.

![image](https://user-images.githubusercontent.com/1120926/61691299-864e7b00-ad2b-11e9-9c8b-9cfbd18a776c.png)

Reply:
No good reason, no. I removed it to get JS syntax highlighting working in my editor and forgot to add it again. Should be fixed now.